tools/seta.py

```diff
@@ -85,7 +95,11 @@ def depth_first(failures, target):
     target = int(total* (target / 100))
     to_remove = []
 
-    to_remove = build_removals(active_jobs, failures, to_remove, target)
+    to_remove, remaining_failures = build_removals(active_jobs, failures, to_remove, target)
+    if ignoreFirstFailure:
+        remaining_jobtypes = [x for x in active_jobs if x not in to_remove]
```

Comment:
please add a comment here outlining the logic we are going to perform in this case.